build.gradle

```diff
@@ -19,6 +19,8 @@ ext {
     greenDaoVersion = "3.2.2"
     loggingInterceptorVersion = '3.6.0'
     okHttpVersion = '3.6.0'
+    gsonVersion = '2.8.0'
```

Comment:
To prevent version conflicts, can we rely on the version used on the imported `ai.elimu:model` library instead?: https://github.com/elimu-ai/model/blob/master/pom.xml#L80

See https://github.com/elimu-ai/appstore/blob/master/app/build.gradle#L49